docs/timezone.md

```diff
@@ -282,23 +285,23 @@ This method returns an array of all the possible exact times that could correspo
 
 If `dateTime` is not a `Temporal.PlainDateTime` object, then it will be converted to one as if it were passed to `Temporal.PlainDateTime.from()`.
 
-Normally there is only one possible exact time corresponding to a wall-clock time, but around a daylight saving change, a wall-clock time may not exist, or the same wall-clock time may exist twice in a row.
+Normally there is only one possible exact time corresponding to a wall-clock time, but around a daylight saving or other political change, a wall-clock time may not exist, or the same wall-clock hour sequence may exist twice in a row (for DST usually just one double-hour).
```

Comment:
What's an "hour sequence" ?  Would be good to understand why "time" isn't OK here. I'd prefer to keep simpler terms when it makes sense.

Reply:
The IANA time zone ["Antarctica/Troll"](https://en.wikipedia.org/wiki/Time_in_Norway#IANA_time_zone_database) has a DST switch of 2 hours, meaning the sequence of local hours can be as follows: <code>22, 23, ***00, 01, 00, 01,*** 02, 03</code>. `00, 01` is an hour sequence and that *sequence* happens twice. The same could possibly happen in the context of permanent UTC offset changes. It shouldn't be thought that it'll definitely always just be one double-hour.

I clarified it a bit. See next commit.

Reply:
My opinion is it'll be really confusing if we use the rare case of a 2-hour DST transition as the example here.

I think the original wording ("same wall-clock time may exist twice in a row") should cover that case as well, or maybe remove "in a row" if you think it too heavily implies "within one 60-minute time span". The same wall-clock time existing twice, seems like it can describe 1-hour DST transitions, 2-hour DST transitions, or time zone boundary changes, as long as they are in the "fall back" direction.

Reply:
I agree with @ptomato. We should try to avoid adding complexity to accommodate rare cases unless it's likely that programmer bugs will result from ignoring those cases. 

What about wording like below, which is shorter, simpler, and makes no assumptions that repeats are one hour long?

> Normally there is only one possible exact time corresponding to a wall-clock time, but around a daylight saving or other offset change, a wall-clock time may not exist, or the same wall-clock time may exist twice.

Reply:
I applied @justingrant's proposal with exception of the word "exactly" in "exactly one possible exact time" (keeping original "only").